.circleci/config.yml

```diff
@@ -6,6 +6,7 @@ jobs:
       xcode: "9.4.1"
     steps:
       - checkout
+      - run: xcrun simctl list
```

Comment:
do we still need this?

Reply:
Since I added `xcrun simctl list` we've miraculously never had the error again.  If I remove it we get the missing simulator error sporadically. This could also help us debug it next time.